frontend/src/pages/CourseEditor/CourseEditorContainer.tsx

```diff
@@ -18,6 +18,8 @@ import { Numbers } from 'enums/numbers';
 import { useSnackbar } from 'notistack';
 import { courseEditorValidationSchema } from 'validations/schemas';
 import { uploadFile } from 'utils/helpers/uploader';
+import { useCallbackPrompt } from 'hooks';
+import ConfirmLeavePage from 'pages/PassingTest/ConfirmLeavePage';
```

Comment:
I guess, it should be reused from components/dialogs. Replace this component to common folder please